packages/atomic/src/components/search/atomic-generated-answer/source-citations.tsx

```diff
@@ -0,0 +1,38 @@
+import {GeneratedAnswerCitation} from '@coveo/headless';
+import {FunctionalComponent, h} from '@stencil/core';
+
+interface SourceCitationsProps {
+  label: string;
+  citations: GeneratedAnswerCitation[];
+  onCitationClick: (citation: GeneratedAnswerCitation) => void;
+}
+
+export const SourceCitations: FunctionalComponent<SourceCitationsProps> = (
+  props
+) =>
+  props.citations.length ? (
+    <div class="source-citations flex">
+      <p class="citations-label text-neutral-dark text-sm shrink-0">
+        {props.label}
+      </p>
+      <div class="citations-container flex items-center flex-wrap">
+        {props.citations.map(
+          (citation: GeneratedAnswerCitation, index: number) => (
+            <a
+              key={citation.id}
+              title={citation.title}
+              href={citation.clickUri ?? citation.uri}
+              target="_blank"
```

Comment:
We should probably add `target="_blank"` to the source of the smart snippet and smart snippet suggestions too.

Make sure to add `rel="noopener"` wherever you have `target="_blank"`.